apps/researcher/src/messages/nl/messages.json

```diff
@@ -150,8 +152,23 @@
     "noEntity": "Community kon niet gevonden worden",
     "editButton": "Community page aanpassen",
     "joinButton": "Word lid van deze community",
-    "objectListsTitle": "Lijsten van objecten door community",
+    "goToListButton": "Bekijk alle objecten",
+    "objectListsTitle": "Lijsten",
+    "objectListsSubTitle": "Deze community heeft {count, plural, =0 {0 lijsten} =1 {1 lijst} other {# lijsten}}",
     "membersTitle": "Leden",
-    "error": "Er is een fout opgetreden bij het ophalen van de community. Probeer het later opnieuw."
+    "error": "Er is een fout opgetreden bij het ophalen van de community. Probeer het later opnieuw.",
+    "addObjectListButton": "Lijst toevoegen"
+  },
+  "AddObjectListForm": {
+    "title": "Lijst toevoegen",
+    "labelName": "Lijstnaam",
+    "labelDescription": "Beschrijving (optioneel)",
+    "labelDescriptionSubTitle": "Geef een beschrijving over deze lijst. Bijvoorbeeld, wat is de geschiedenis van deze lijst. Vertel iets over het soort objecten.",
+    "buttonSubmit": "Lijst aanmaken",
+    "buttonCancel": "Annuleren",
+    "nameTooSmall": "Lijstnaam is vereist",
+    "nameTooBig": "Lijstnaam mag maximaal 255 tekens bevatten",
```

Comment:
Thinking out load: 255 is a restriction of a data type of the database, but from a user perspective this could sound weird: why 255 chars - why not 251 or 260? Can we use a 'sensible' max instead, e.g. 250?